pkg/server/builds/service.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package builds
+
+import (
+	"github.com/daytonaio/daytona/internal/util"
+	"github.com/daytonaio/daytona/pkg/build"
+	"github.com/docker/docker/pkg/stringid"
+)
+
+type IBuildService interface {
+	Create(*build.Build) error
+	Find(filter *build.Filter) (*build.Build, error)
+	List(filter *build.Filter) ([]*build.Build, error)
+	Delete(id string) error
+}
+
+type BuildServiceConfig struct {
+	BuildStore build.Store
+}
+
+type BuildService struct {
+	buildStore build.Store
+}
+
+func NewBuildService(config BuildServiceConfig) IBuildService {
+	return &BuildService{
+		buildStore: config.BuildStore,
+	}
+}
+
+func (s *BuildService) Create(b *build.Build) error {
```

Comment:
`build.Build` is a bit too "verbose" here since we modify 3 properties always. I suggest we either have a create DTO or pass props as arguments if there are 2-3 of them